examples/python/partial_md_example.py

```
@@ -143,7 +146,7 @@ def invalidate_target_metadata(
     assert target_agent.register_memory(target_reg_descs2) is not None
 
     # Default port for initiator
-    agent_config2 = nixl_agent_config(True, True, init_port)
+    agent_config2 = nixl_agent_config(True, enable_listen, init_port)
```

Ruff (0.15.2)

[error] Boolean positional value in function call

(FBT003)